source/ranges.tex

```diff
@@ -5248,8 +5245,7 @@
 
     constexpr auto begin();
 
-    constexpr auto end()
-    { return ranges::end(@\exposid{base_}@); }
+    constexpr auto end() { return ranges::end(@\exposid{base_}@); }
```

Comment:
I don't think I like a non-empty definition on the same line as the "declaration" part.